File: examples/corpus/ts/01_pow_mod.before.ts
function calculatePower(base: number, exponent: number): number {
  let result = 1;
  for (let i = 0; i < exponent; i++) { // iterate exponent times
    result = result * base;
  }
  return result;
}

class MathUtility {
  compute(x: number): number {
    return x * x;
  }
}

const MOD = 1000000007;

function processData(data: number[]): number {
  try {
    let ans = 0;
    for (let i = 0; i < data.length; i++) {
      ans = (ans + data[i]) % MOD;
    }
    return ans;
  } catch (e) {
    console.log("An error occurred");
    return -1;
  }
}

File: examples/corpus/py/01_pow_mod.before.py
def calculate_power(base, exponent):
    result = 1
    for i in range(exponent):  # iterate exponent times
        result = result * base
    return result

class MathUtility:
    def compute(self, x):
        return x * x

MOD = 1000000007

def process(data):
    try:
        ans = 0
        for i in range(len(data)):
            ans = (ans + data[i]) % MOD
        return ans
    except Exception as e:
        print("An error occurred")
        return -1
